### DISCUSSION TITLE
Custom fields using @CkEditor and genxml/edt/txtedit4 Paths - loading but not saving.

### DISCUSSION THREAD

**Discussion by matt-binary-synergy** [openstore-ecommerce/OpenStore#279] [Q&A] [answered]
I'm trying to add some custom fields we had in a previous version of nbstore, they were using paths like;
`genxml/lang/genxml/edt/txtedit1 `
`genxml/lang/genxml/edt/txtedit2 `
etc

In the new Admin_ProductDetail.cshtml (also tried productfields.cshtml) I have copied the block for the description field and changed the names and path to the above and the field does load and show correctly! 
However its not being saved when edited.

@Textbox with "`genxml/lang/genxml/textbox/txtedit10`" style paths work and
@CheckBox with paths of "`genxml/checkbox/chksamplesize30_n`" work ok too.

Having narrowed this down, its not the CkEditor part its the paths with "`/edt/`" in.
Can you point me to where these custom fields are saved in source? 
I would rather not have to try and translate in sql if possible.

**Comment by matt-binary-synergy** [comment, marked as answer]
Never mind found the answer, simply change the path! to /textbox/....
Now sure how it worked but it did... it found the field source under the different path and saved it...
not sure how to close these discussions but please do....

**Comment by DNNMonster** [comment]
Glad you solved the xml path issue.  Once you get it right you'll likely never have to look at it again.  Github doesn't let you close a discussion but it should have your post marked as the answer.

**Comment by leedavi** [comment]
FYI.  The  "genxml/edt/.." xpath was a legacy from NBS v2.  I think in v4 it was removed.
